api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -209,4 +217,11 @@ message ExtProcOverrides {
 
   // Set a different gRPC service for this route than the default.
   config.core.v3.GrpcService grpc_service = 5;
+  
+  // [#not-implemented-hide:]
+  // The Metadata field can be used to provide additional information
+  // about the external processing filter. It can be used for configuration,
+  // stats, and logging. The metadata should go under the filter namespace
+  // that will need it.
+  core.v3.Metadata metadata = 6;
```

Comment:
I think you need to have `config.core.v3.Metadata` here.
because that is under `config` space and here is `extensions` space